site/content/contribute/containers/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Containers"
+section: "contribute"
+---
+
+# Mattermost Containers
+
+Mattermost uses the [Docker Registry](https://hub.docker.com/u/mattermost) to publish the official images for the Mattermost Server and also for other supporting images that is used for internal/public development and testing
+
+This page will list all the Docker Repositories used by Mattermost and community
+
+- [mattermost/mattermost-enterprise-edition](https://hub.docker.com/r/mattermost/mattermost-enterprise-edition) - **Official Mattermost Server** image for the **Enterprise edition version**. To find the Dockerfile please refer to https://github.com/mattermost/mattermost-server/tree/master/build
+
+- [mattermost/mattermost-team-edition](https://hub.docker.com/r/mattermost/mattermost-team-edition) - **Official Mattermost Server** image for the **Team edition version**. To find the Dockerfile please refer to https://github.com/mattermost/mattermost-server/tree/master/build
+
+- [mattermost/mattermost-push-proxy](https://hub.docker.com/r/mattermost/mattermost-push-proxy) - Mattermost Push Proxy. Documentation: https://developers.mattermost.com/contribute/mobile/push-notifications/service/ GitHub: https://github.com/mattermost/mattermost-push-proxy
+
+- [mattermost/mattermost-operator](https://hub.docker.com/r/mattermost/mattermost-operator) - Official image for Mattermost Operator for Kubernetes. For more information: https://github.com/mattermost/mattermost-operator
+
+- [mattermost/mattermost-test-enterprise](https://hub.docker.com/r/mattermost/mattermost-test-enterprise) - Repository where all testing images is publish for any type of testing. Those images are build from the CircleCI Pipelines from the [mattermost-server](https://github.com/mattermost/mattermost-server) and [mattermost-webapp](https://github.com/mattermost/mattermost-webapp)
+
+- [mattermost/mattermost-test-team](https://hub.docker.com/r/mattermost/mattermost-test-team) - Repository where all testing images is publish for any type of testing. Those images are build from the CircleCI Pipelines from the [mattermost-server](https://github.com/mattermost/mattermost-server) and [mattermost-webapp](https://github.com/mattermost/mattermost-webapp)
+
+- [mattermost/mattermost-prod-app](https://hub.docker.com/r/mattermost/mattermost-prod-app) - Community driven image for Mattermost Server. **This Docker repository will be deprecated in Mattermost 6.0**. For more information and to check the dockerfile please refer to https://github.com/mattermost/mattermost-docker
+
+- [mattermost/mattermost-prod-db](https://hub.docker.com/r/mattermost/mattermost-prod-db) - Community driven image for Database to run together with **mattermost/mattermost-prod-app**. **This Docker repository will be deprecated in Mattermost 6.0**. For more information and to check the dockerfile please refer to https://github.com/mattermost/mattermost-docker
```

Comment:
```suggestion
- [mattermost/mattermost-prod-db](https://hub.docker.com/r/mattermost/mattermost-prod-db) - Community driven image for Database to run together with **mattermost/mattermost-prod-app**. 

**This Docker repository will be deprecated in Mattermost 6.0**. For more information and to check the Dockerfile please refer to https://github.com/mattermost/mattermost-docker.
```

Reply:
If we move it to its own line, it'll break the bulleted list. I worry it won't be clear which of the images is deprecated in 6.0 (the one above or below this deprecation notice)

Reply:
Ah, that makes sense. Fixed it but kept the capitalization from dockerfile > Dockerfile.